spec/unit/device-converter.spec.ts

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { DeviceInfo, DeviceVerification } from "../../src/crypto/deviceinfo";
+import { DeviceKeys } from "../../src";
+import { downloadDeviceToJsDevice } from "../../src/rust-crypto/device-converter";
+import { deviceInfoToDevice } from "../../src/crypto/device-converter";
+
+describe("device-converter", () => {
```

Comment:
please could you separate the tests for the rust `device-converter` and the legacy one into separate files?